src/UpdatedPackagesCollector.js

```diff
@@ -52,52 +52,45 @@ export default class UpdatedPackagesCollector {
   collectUpdatedPackages() {
     this.logger.info("", "Checking for updated packages...");
 
-    const hasTags = GitUtilities.hasTags(this.execOpts);
+    const { execOpts, options } = this;
+    const { canary } = options;
+    let { since } = options;
 
-    if (hasTags) {
-      const tag = GitUtilities.getLastTag(this.execOpts);
-      this.logger.info("", "Comparing with tag " + tag);
-    } else {
-      this.logger.warn("", "No tags found!");
-      this.logger.info("", "Comparing with initial commit.");
-    }
+    if (GitUtilities.hasTags(execOpts)) {
+      if (canary) {
+        let currentSHA;
 
-    let commits;
-
-    if (this.options.canary) {
-      let currentSHA;
+        if (canary !== true) {
+          currentSHA = canary;
+        } else {
+          currentSHA = GitUtilities.getCurrentSHA(execOpts);
+        }
 
-      if (this.options.canary !== true) {
-        currentSHA = this.options.canary;
-      } else {
-        currentSHA = GitUtilities.getCurrentSHA(this.execOpts);
+        since = this.getAssociatedCommits(currentSHA);
+      } else if (!since) {
+        since = GitUtilities.getLastTag(execOpts);
       }
-
-      commits = this.getAssociatedCommits(currentSHA);
-    } else if (hasTags) {
-      commits = GitUtilities.describeTag(
-        GitUtilities.getLastTaggedCommitInBranch(this.execOpts),
-        this.execOpts
-      );
     }
 
+    this.logger.info("", `Comparing with ${since || "initial commit"}.`);
```

Comment:
@evocateur (mentioning since I want to place special emphasis here), I removed the warning if no tags exist because it doesn't really feel like a warning, especially with the addition of `--since` as an empty arg is implicitly the latest tag (or initial commit).

If there's any contention here, or it's a breaking change, I'm more than happy to add it back in.